src/test-utils/dom/flashbar/index.ts

```diff
@@ -31,6 +31,20 @@ export default class FlashbarWrapper extends ComponentWrapper {
     );
   }
 
+  /**
+   * Returns the wrapper of the first flash list item that matches the specified test ID.
+   * If the items are stacked, the hidden items will not be returned.
+   *
+   * Looks for the `data-testid` attribute that is assigned via `items` prop.
+   * If no matching flash list item is found, returns `null`.
+   *
+   * @param {string} testId
+   * @returns {FlashbarWrapper | null}
+   */
+  findItemByTestId(testId: string): FlashWrapper | null {
+    return this.findComponent(`.${styles['flash-list-item']}[data-testid="${CSS.escape(testId)}"]`, FlashWrapper);
```

Comment:
Existing similar places use `escapeSelector` from test-utils (https://github.com/cloudscape-design/test-utils/blob/f49a5a90ce077253f74d82659f1cb6e83092a2a4/src/core/utils.ts#L7), which ensures that this is polyfilled. Might be good to use that here too for consistency

Reply:
Good point! Replaced it. Also replaced one more occurrence that we had used this before!